src/spaces/graph.jl

```diff
@@ -91,7 +90,13 @@ nodes(model::ABM{<:AbstractAgent,<:GraphSpace}) = 1:nv(model)
 #######################################################################################
 function space_neighbors(pos::Int, model::ABM{A,<:GraphSpace}, args...; kwargs...) where {A}
     nn = node_neighbors(pos, model, args...; kwargs...)
-    Iterators.flatten(model.space.s[i] for i in Iterators.flatten((pos,nn)))
+    # This call is faster than reduce(vcat, ..), or Iterators.flatten
+    vcat(model.space.s[pos], model.space.s[nn]...)
```

Comment:
Perhaps
```julia
x = copy(model.space.s[pos])
for z in nn
append!(x, model.space.s[z]
end
return x
```
is the fastest version...? I don't really know the performance caveats of `vcat` (wtherhr it performns well for many arguments0

Reply:
Nope, `vcat` is faster. 
Commited version:
```julia
@btime space_neighbors($pos, $model, 20)
  58.742 μs (589 allocations: 147.55 KiB)

@btime space_neighbors($pos, $model, 100)
  1.950 ms (10328 allocations: 9.44 MiB)
```
Append version:
```julia
@btime space_neighbors2($pos, $model, 20)
  61.719 μs (632 allocations: 149.75 KiB)

@btime space_neighbors2($pos, $model, 100)
  2.066 ms (10490 allocations: 9.45 MiB)
```